.circleci/config.yml

```diff
@@ -210,26 +210,48 @@ jobs:
       - android/accept-licenses
       - android-wordpress-orb/restore-gradle-cache
       - android/restore-build-cache
-      - run:
-          name: Replace API_KEY
-          working_directory: examples/purchase-tester
-          command: |
-            sed -i s/api_key/$API_KEY/ src/main/java/com/revenuecat/purchasetester/MainApplication.kt
       - run:
           name: Prepare Keystore
           working_directory: examples/purchase-tester
           command: echo $RELEASE_KEYSTORE | base64 -d > keystore
       - run:
           name: Assemble
           command: |
-            ./gradlew -p examples/purchase-tester assemble \
+            ./gradlew -p examples/purchase-tester assembleLatestDependenciesRelease \
```

Comment:
If there's a compilation error in the unityIAP version, it would be caught here right? Or is there any other place where we build that version and catch compilation errors?

Reply:
Hmm I believe it won't... You have a point there, It's probably a good idea to just take a little longer and build the app on the different variants to make sure the app compiles in all variants. We do have the API tests that build in all variants to check for most issues, but I don't think it would hurt to also build the purchase tester app.